File: java-service/src/main/java/com/analyticore/analyseservice/domain/JobStatus.java
package com.analyticore.analyseservice.domain;

/**
 * Capa de Dominio: estados del ciclo de vida de un trabajo.
 * Compartido conceptualmente con python-service/domain/models.py
 */
public enum JobStatus {
    PENDIENTE,
    PROCESANDO,
    COMPLETADO
}

File: frontend/src/domain/jobStatus.js
/**
 * Capa de Dominio — Estados del ciclo de vida de un job.
 * Sincronizado con python-service/domain/models.py y PostgreSQL columna 'status'.
 */
export const JobStatus = {
  PENDIENTE: 'PENDIENTE',
  PROCESANDO: 'PROCESANDO',
  COMPLETADO: 'COMPLETADO',
}

export const POLL_INTERVAL_MS = 1000
